CONTRIBUTORS

```diff
@@ -30,3 +30,4 @@ Mike Playle <mike@mythik.co.uk>
 Phil Elson <pelson.pub@gmail.com>
 Robert Coup <robert.coup@koordinates.com>
 Tiago Brito <tiago.brito@90poe.io>
+Zachary Burnett <zachary.r.burnett@gmail.com>
```

Comment:
It looks like we were doing it wrong before, but I think you should also be in the AUTHORS file.

Reply:
You're still not in the AUTHORS file, but I'll fix it later with the others.